php_phongo.c

```diff
@@ -342,7 +341,6 @@ bool phongo_query_init(php_phongo_query_t *query, zval *filter, zval *options TS
 				return false;
 			}
 
-			convert_to_array_ex(&projection);
 			query->selector = bson_new();
```

Comment:
How is this related to $sort ?
Is it similar issue, in which case, should it not be convert_to_object_ex() then?